app/Example.java

```diff
@@ -0,0 +1,50 @@
+import android.view.View;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Message;
+import io.ably.lib.types.Param;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
+
+	public void publishAndSubcscribe(Button buttonPublish, Button buttonHistory) {
```

Comment:
Typo, publishAndSubscribe.  Also, shouldn't this be called `setupPublishAndSubscribe` to be clearer?  Why is this not just in an initializer or something like that?